packages/checkout-button-integration/src/CheckoutButton.tsx

```diff
@@ -12,19 +12,49 @@ const CheckoutButton: FunctionComponent<CheckoutButtonProps> = ({
     methodId,
     onUnhandledError,
 }) => {
+    const beautifyCheckoutButtons = useCallback((): void => {
+        if (methodId === 'amazonpay') {
+            if (!document.querySelector('.checkout-button-container')) {
+                return;
+            }
+
+            const container: HTMLElement | null = document.querySelector(
+                '#amazonpayCheckoutButton > div',
+            );
+
+            if (!container || !container.shadowRoot) return;
+
+            const amazonPayButton: HTMLElement | null =
+                container.shadowRoot.querySelector('.amazonpay-button-view1');
+
+            if (amazonPayButton) {
+                amazonPayButton.style.height = '36px';
+            }
```

Comment:
It looks like we are changing `.amazonpay-button-view1` twice here and the file below. Is this intentional?
Also, the brutal method of beautifiying a button is really not ideal. Can we just keep it inside the button file, not this shared component?

Reply:
Agree with Peng here

Reply:
Building an Amazon package, as Animesh recommended, should be the best practice.
The v1 button appears in the customer step rather than at the top of the checkout page. As a result, we ought to focus on V2 button design right now because these are the ones we want merchants to use.